Article/Article.js

```diff
@@ -3,18 +3,31 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+    // STRETCH 3: Button to remove article marked as read
+    this.readButton = this.domElement.querySelector('.readButton');
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Click to Expand';
+    // STRETCH 3: Click handler to remove article marked as read
+    this.readButton.addEventListener('click', this.readArticle.bind(this));
```

Comment:
this works and its always just preference, but dont forget you can also implicitly bind this with an arrow function
```
    this.readButton.addEventListener('click', () => this.readArticle());
```